### ISSUE TITLE
Custom Terms of Service and Privacy Policy links not displayed in about menu nor anywhere else after login

### ISSUE THREAD

**Issue by Touchwoody** [mattermost/mattermost#14531]
#### Summary
When running an own mattermost (web) server one needs to display or link to their own privacy policy somewhere on the web pages. This is often by law: the user has to be able to find these information in an easy, visible and unblocked way. 
In the past, these were at least displayed via the "about" link in the main menu.

Originally I wanted to file the same error as in https://mattermost.atlassian.net/browse/MM-24816 but it seems this goes deeper. 
Obviously there was a design decision made in https://mattermost.atlassian.net/browse/MM-19759, but this was flawed resp. incomplete. After changing this behavior of the about screen, now the terms of service and  privacy policy link are not shown anywhere. This is bad.

I urge you to reconsider the decision made in MM-19759 or to offer some other means to properly display the _site-local_ ToS and esp. the privacy policy to the user. 

I am not sure what lead to the decision to hardwire to mattermosts own ToU and PP in the about screen. 
In my mind, as that is always a site-local server installation, the links there should of course point to the ToU/PP that apply to that site (the about screen is even titled with config.json variable "SiteName").

#### Steps to reproduce

Use the 'About' link in main menu. An "About" screen will pop up with links to "Terms of Service - Privacy Policy". These are not linked to the site local ToS/PP but to mattermosts ToS/PP. 

Nowhere else on the web page or menus is a link to the site-local ToS/PP. 

#### Expected behavior

A user needs to find links to the site-local terms of service and privacy policy somewhere. 

Most obvious place would be Main Menu-> About.

#### Observed behavior (that appears unintentional) 

Mattermost own terms of use and privacy policy are linked on "about" screen. config.json settings "TermsOfServiceLink" and "PrivacyPolicyLink" are ignored.

There is no link displayed to the custom site-local terms of service and privacy policy anywhere else on the web page.

#### Possible fixes

Undo [MM-19759](https://mattermost.atlassian.net/browse/MM-19759) or display local terms of service and privacy policy links elsewhere on the web pages, e.g. in the hamburger menu or maybe a web page footer (which would be quite common).

**Comment by amyblais** [comment]
I've asked our product manager to review this,

**Comment by amyblais** [comment]
@Touchwoody Our PMs clarified the docs for this: https://docs.mattermost.com/administration/config-settings.html#terms-of-service-link. Let me know if this helps,

**Comment by Touchwoody** [comment]
Well, that clarifies the current behaviour, but doesn't solve the problem. 

Everyone hosting mattermost CE needs a way to show it's _own_ custom privacy policy and terms of service to it's users, too. Obviously since 5.17 there are no more means to do so. This is fatal. 

As mentioned, it is often by law, that PP and ToU have to be easy to find for the user. At least in Germany it is. Right now the link is only visible in the login screen and you seriously can't expect the user to log out (!) before he is able to find/read the PP/ToU.

So what is desperately needed, is a link to the custom PP/ToU "somewhere" in the menu or the webpage, either by undoing MM-19759 or by adding the link somewhere else.

**Comment by aaronrothschild** [comment]
@Touchwoody I'm not the PM for this particular area - but I'm wondering if as an interim solution to this issue could be to use the Welcomebot plugin.  It could send a message to every new user and in that customizable message, you could welcome them to the community and offer a link to the custom ToS that would remain in that bot channel for them to go back and refer to.  I'm not sure if that satisfies the policy requirement fully, but may help to mitigate this issue.

**Comment by wiersgallak** [comment]
@Touchwoody Thank you for the feedback.  We have created a ticket to add the information back to the About Mattermost modal that is accessible from the main menu. https://mattermost.atlassian.net/browse/MM-26294 Please let us know if you think this solution does not meet your requirements.

**Comment by Touchwoody** [comment]
Sorry for the late response:
Yes, displaying both -- mattermost _and_ custom PP/ToU --  in the "about" window sounds very reasonable.

**Comment by RouL** [comment]
Could somebody clarify why on earth the "original Links" should be shown at all?

If you're running a self-hosted platform it is totally confusing to your users to find a link to ToS and PP not related at all to that platform. It's even worse, if the language of those is not their mother tongue. A platform should only show it's very own ToS and PP, it's a total no-brainer.

I'm working for one of the biggest welfare organizations in Germany and we're trying to roll out MM for our 3000 member organizations and this is a real problem. The worst thing is the PP. The ToS is at least on the MM website, which is already harming the trust in the platform, since it's a third party website, but the PP is hosted on GitHub. I am a developer myself, but put yourself in the shoes of a simple user, clicking on what looks like the link to the PP of the platform and then landing on GitHub. Seriously folks, what were you thinking? That is OK for technical users like us, but not for a simple non-technical user using such a platform.

Don't mean to offend anybody, but this hat to be said.

PS: Love your work. :)

**Comment by RouL** [comment]
Anybody from Mattermost still looking into this?

**Comment by amyblais** [comment]
We have a ticket to add the information back to the About Mattermost modal that is accessible from the main menu https://mattermost.atlassian.net/browse/MM-26294 Please let us know if you think this solution does not meet your requirements.

**Comment by lieut-data** [comment]
Thanks for reporting this issue and for the community discussion around it. After multiple years of inactivity, we're closing this to help keep our issue tracker focused on active work. If this is still relevant, please feel free to open a new issue with updated context and we'll be happy to take another look.
